apps/emqx/test/emqx_crl_cache_SUITE.erl

```diff
@@ -52,6 +52,7 @@ init_per_testcase(TestCase, Config) when
     ServerPid = start_crl_server(CRLPem),
     IsCached = lists:member(TestCase, [t_filled_cache, t_revoked]),
     Apps = start_emqx_with_crl_cache(#{is_cached => IsCached}, TestCase, Config),
+    emqx_logger:set_log_level(debug),
```

Comment:
Forgotten?

Reply:
No.  The idea is that, if this test fails again, we'll have [this log](https://github.com/emqx/emqtt/blob/5a93b9c74106d10131a4d1d48f872b604aae3cc7/src/emqtt.erl#L1049) available.  Granted, it's not changed back after the test...  But hopefully that's not too much of a problem?

Reply:
Ha.  Indeed, one test in another suite broke because of the log level...  :melting_face: 

Removed.